career_skill_atlas/urls.py

```diff
@@ -15,8 +15,16 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
+
+api_patterns = [
+    path('users/', include(('user.urls', 'user'))),
+]
 
 urlpatterns = [
+
+    #ToDo(Vita): clarify, do we need prefix for API urls, like: path('api/v1/', include(api_patterns)),
+    path('', include(api_patterns)),
     path('admin/', admin.site.urls),
+    path('api-auth/', include('rest_framework.urls', namespace='rest_framework')),
```

Comment:
why do we need it?

Reply:
this is from drf ofical tutorial to get posibility to make requests from DRF UI, which requare auth. Probably will need it future

Reply:
@vitayushchyk Ivan already implemented endpoints for auth, we don't have other endpoints according architecture, so please drop it